src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Inputs/BitTextFieldTests.cs

```diff
@@ -203,33 +203,33 @@ public void BitTextFieldMustRespondToTheKeyEvent(bool isEnabled, bool IsMultilin
 
             var bitTextField = IsMultiline ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
 
-            bitTextField.KeyDown("down");
-            Assert.AreEqual(isEnabled ? 1 : 0, component.Instance.KeyDownedValue);
+            bitTextField.KeyDown(keyDownValue);
+            Assert.AreEqual(isEnabled ? keyDownValue : null, component.Instance.KeyDownedValue);
 
-            bitTextField.KeyUp("up");
-            Assert.AreEqual(isEnabled ? 1 : 0, component.Instance.KeyUpedValue);
+            bitTextField.KeyUp(keyUpValue);
+            Assert.AreEqual(isEnabled ? keyUpValue : null, component.Instance.KeyUpedValue);
         }
 
-       // [DataTestMethod,
-       //    DataRow(true, false),
-       //    DataRow(true, true),
-       //    DataRow(false, false),
-       //    DataRow(false, true),
-       //]
-       // public void BitTextFieldMustRespondToTheChangeEvent(bool isEnabled, bool IsMultiline)
-       // {
-       //     var component = RenderComponent<BitTextFieldTest>(
-       //         parameters =>
-       //         {
-       //             parameters.Add(p => p.IsEnabled, isEnabled);
-       //             parameters.Add(p => p.IsMultiLine, IsMultiline);
-       //         });
-
-       //     var bitTextField = IsMultiline ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
-
-       //     bitTextField.KeyDown("a");
-
-       //     Assert.AreEqual(isEnabled ? 1 : 0, component.Instance.CurrentCount);
-       // }
+        [DataTestMethod,
+           DataRow(true, false),
+           DataRow(true, true),
+           DataRow(false, false),
+           DataRow(false, true),
+       ]
+        public void BitTextFieldMustRespondToTheChangeEvent(bool isEnabled, bool IsMultiline)
+        {
+            var component = RenderComponent<BitTextFieldTest>(
+                parameters =>
+                {
+                    parameters.Add(p => p.IsEnabled, isEnabled);
+                    parameters.Add(p => p.IsMultiLine, IsMultiline);
+                });
+
+            var bitTextField = IsMultiline ? component.Find(".bit-txt textarea") : component.Find(".bit-txt input");
+
+            bitTextField.KeyDown("a");
+
+            //Assert.AreEqual(isEnabled ? 1 : 0, component.Instance.CurrentCount);
```

Comment:
please add a TODO comment to explain why these lines are commented, just like the ones commented before because of issues in the Bunit framework.